webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -41,15 +41,15 @@ export const getNotifications = url => dispatch => {
   );
 };
 
-export const onMarkAsRead = (group, id) => dispatch => {
+export const onMarkAsRead = (group, ids) => dispatch => {
   dispatch({
     type: NOTIFICATIONS_MARK_AS_READ,
     payload: {
       group,
-      id
+      ids
     }
   });
-  API.markNotificationAsRead(id);
+  ids.forEach(id => { API.markNotificationAsRead(id); });
```

Comment:
i think it would be more efficient to have an api method accepting an array of IDS, instead of making many seperate calls